core/invehicle-digital-twin/src/main.rs

```diff
@@ -72,30 +69,26 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // Load the config.
     let settings = invehicle_digital_twin_config::load_settings();
     let invehicle_digital_twin_authority = settings.invehicle_digital_twin_authority;
-    let chariott_url_option = settings.chariott_url;
+    let chariott_uri_option = settings.chariott_uri;
 
     // Setup the HTTP server.
     let addr: SocketAddr = invehicle_digital_twin_authority.parse()?;
-    let digitaltwin_impl = digitaltwin_impl::DigitalTwinImpl {
+    let invehicle_digital_twin_impl = invehicle_digital_twin_impl::InvehicleDigitalTwinImpl {
         entity_access_info_map: Arc::new(RwLock::new(HashMap::new())),
     };
     let invehicle_digital_twin_address = format!("http://{invehicle_digital_twin_authority}"); // Devskim: ignore DS137138
-    let invehicle_digital_twin_url = Url::parse(&invehicle_digital_twin_address)?;
-    let providerservice_impl =
-        providerservice_impl::ProviderServiceImpl::new(invehicle_digital_twin_url);
     let server_future = Server::builder()
-        .add_service(DigitalTwinServer::new(digitaltwin_impl))
-        .add_service(ProviderServiceServer::new(providerservice_impl))
+        .add_service(InvehicleDigitalTwinServer::new(invehicle_digital_twin_impl))
         .serve(addr);
     info!("The HTTP server is listening on address '{invehicle_digital_twin_address}'");
 
-    // Register the digital twin service with Chariott if Chariott's URL was provided in the config.
+    // Register the digital twin service with Chariott if Chariott's URI was provided in the config.
```

Comment:
nit: Seems like you are changing it elsewhere, add `invehicle` before `digital twin`

Reply:
Fixed.